build/js/live-editor.tooltips.js

```diff
@@ -1483,7 +1483,8 @@ window.TooltipEngine = Backbone.View.extend({
             return false;
         }
         if (this.isWithinComment(params.pre)) {
-            return false;
+            // if within comment, emit default event to hide existing tooltip
+            params = this.requestTooltipDefaultCallback;
```

Comment:
This fixes the issue where, when a tooltip is showing, if the user clicks on a commented line of code, the displayed tooltip doesn't disappear.
Returning false prevents another tooltip from being displayed, but doesn't do anything to hide the existing tooltip.

Originally I had two lines here, directly from the code which ends up being called when emitting this event (see lones 1433-1440).

Do you think it would be better instead to stick with the original idea and simply paste these lines in place of the line of code in question?

```
this.currentTooltip.$el.hide();
this.currentTooltip = undefined;
```